upi/openstack/02_network.yaml

```diff
@@ -35,6 +35,20 @@
       name: "{{ os_subnet }}"
       tags: "{{ [cluster_id_tag] }}"
 
+  - name: 'Create pods subnet pool'
+    os_create_subnet_pool:
+      name: "{{ subnet_pool }}"
```

Comment:
where is this defined?

Reply:
In here[1]. This was already defined since last UPI support got merged, as we needed to delete the subnet pool created by Kuryr. 

[1] https://github.com/MaysaMacedo/installer/blob/ensure-subnet-pool/upi/openstack/common.yaml#L31